linux/CMakeLists.txt

```
@@ -1,19 +1,22 @@
 #icons
-## application icon
+set(sizes 8x8 16x16 22x22 24x24 32x32 36x36 42x42 48x48 64x64 72x72 80x80 96x96 128x128 192x192 256x256 512x512)
+
+# Install application icon
 install(FILES ${CMAKE_SOURCE_DIR}/images/icons/qgis_icon.svg RENAME qgis.svg DESTINATION share/icons/hicolor/scalable/apps)
-install(FILES ${CMAKE_SOURCE_DIR}/images/icons/qgis-icon-16x16.png RENAME qgis.png DESTINATION share/icons/hicolor/16x16/apps)
-install(FILES ${CMAKE_SOURCE_DIR}/images/icons/qgis-icon-512x512.png RENAME qgis.png DESTINATION share/icons/hicolor/512x512/apps)
+foreach(size ${sizes})
+  install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/icons/qgis-icon${size}.png RENAME qgis.png DESTINATION share/icons/hicolor/${size}/apps/)
```

Am I wrong but it should be: (lack of a '-' before size)

```suggestion
  install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/icons/qgis-icon-${size}.png RENAME qgis.png DESTINATION share/icons/hicolor/${size}/apps/)
```

---

No hyphen before size e.g.:
https://github.com/qgis/QGIS/blob/master/linux/icons/qgis-icon128x128.png

---

ok, tnx